commcare_connect/conftest.py

```diff
@@ -145,3 +146,39 @@ def program_manager_org_user_member(program_manager_org) -> User:
 @pytest.fixture
 def program_manager_org_user_admin(program_manager_org) -> User:
     return program_manager_org.memberships.filter(role="admin").first().user
+
+
+@pytest.fixture
+def check_basic_permissions(client):
```

Comment:
Curious why you decided to do this as a fixture, rather than just a function in the test file. We don't have any other "test functions" as fixtures, just objects.